### DISCUSSION TITLE
Transferring kickstart-nix.nvim

### DISCUSSION THREAD

**Discussion by mrcjkb** [nix-community/infra#1193] [Q&A] [answered]
Hey 👋 

I've been wondering if it would make sense to transfer [kickstart-nix.nvim](https://github.com/mrcjkb/kickstart-nix.nvim) to nix-community?

**Comment by Mic92** [comment, marked as answer]
I think it make sense. I sent you an invite. You can join by visiting: https://github.com/nix-community
Let me know when you moved the repository so I can do some post-migration steps i.e. giving you admin rights back.

**Reply by mrcjkb** [reply]
Done 😃

**Reply by Mic92** [reply]
There is now a team that you can invite more contributor to: https://github.com/orgs/nix-community/teams/kickstart-nix-nvim/members
